bigbluebutton-html5/imports/ui/components/media/webcam-draggable-overlay/context.jsx

```diff
@@ -99,7 +99,7 @@ const reducer = (state, action) => {
       };
     }
     default: {
-      throw new Error('Unexpected action');
+      throw new Error(`Unexpected action: ${JSON.stringify(action)}`);
```

Comment:
Question: do we know if that stringify is always safe (ie won't choke on circular objects?). Asking because I have a bit of trouble on getting a grasp at the action object structure (and whether it's always consistent). Would be cool to extract the useful attributes and concatenate them to the error message instead.